packages/dmn-editor/tests/e2e/__fixtures__/base.ts

```diff
@@ -32,6 +37,11 @@ type DmnEditorFixtures = {
   jsonModel: JsonModel;
   nodes: Nodes;
   palette: Palette;
+  generalProperties: GeneralProperties;
+  decisionProperties: DecisionProperties;
+  knowledgeSourceProperties: KnowledgeSourceProperties;
+  propertiesPanel: PropertiesPanelBase;
+  generalDecisionServiceProperties: GeneralDecisionServiceProperties;
```

Comment:
Looking at this, I don't think a consistency between the names.


```suggestion
  generalPropertiesPanel: GeneralPropertiesPanel;
  decisionPropertiesPanel: DecisionPropertiesPanel;
  knowledgeSourcePropertiesPanel: KnowledgeSourcePropertiesPanel;
  propertiesPanel: PropertiesPanel;
  decisionServicePropertiesPanel: DecisionServicePropertiesPanel;
```

Also, what is `generalProperties`? If we want to use this class as a base for the others, we still need to create the specific ones. After we have the specific, we wouldn't need to directly use the "GeneralProperties" class.

From this, I can see it's missing a few panels:
- TextAnnotation
- Group
- InputData
- BKM
- Global (when no node is selected)
- Multi (when multiple nodes are selected)

Reply:
Incorporated as part of #27